spyder_terminal/config.py

```diff
@@ -20,6 +20,14 @@
       'shell': 'cmd' if WINDOWS else 'bash'
      }
      ),
+    ('shortcuts',
+     {
+      'terminal/copy': 'Ctrl+Alt+C',
```

Comment:
All other shortcuts are working pretty well, with the exception of this one. It's not only copying text but also introducing `^C` in the prompt on Linux:

![Selección_103](https://user-images.githubusercontent.com/365293/72861601-70991e80-3c98-11ea-8e37-5d000261453a.png)

To avoid that, let's change this one to `Ctrl+Alt+Shift+C`, but only for Linux. And for consistency, please change paste to `Ctrl+Alt+Shift+V` (only for Linux too).

That's a compromise, but not a bad one, because I think users on Linux barely resort to those shortcuts and use instead Xorg clipboard instead on the terminal (i.e. copying by selecting something with the mouse and pasting with a mouse middle-click).